Pod/Classes/TMPodspec.m

```diff
@@ -61,21 +76,125 @@ - (NSString *)version {
     return [self stringForKey:PodspecKeyVersion];
 }
 
-- (NSString *)summary {
-    return [self stringForKey:PodspecKeySummary];
+- (NSArray *)authors {
+    NSString *authorName = [self stringForKey:PodspecKeyAuthor];
+    NSArray *authorNames = [self arrayForKey:PodspecKeyAuthors];
+    NSDictionary *authorNamesAndAddresses = [self dictionaryForKey:PodspecKeyAuthors];
```

Comment:
Not necessary to do now but might be worth doing stuff like these in individual classes, like have a class that takes `self.dictionary` and returns an authors array, another that takes `self.dictionary` and returns a screenshots array, etc.